browser/brave_profile_prefs.cc

```diff
@@ -129,10 +129,13 @@
 #if !BUILDFLAG(IS_ANDROID)
 #include "brave/browser/search_engines/search_engine_provider_util.h"
 #include "brave/browser/ui/startup/default_brave_browser_prompt.h"
-#include "brave/browser/ui/views/tabs/brave_tab_prefs.h"
 #include "brave/components/brave_private_new_tab_ui/common/pref_names.h"
 #endif
 
+#if defined(TOOLKIT_VIEWS)
+#include "brave/browser/ui/views/tabs/brave_tab_prefs.h"
```

Comment:
This was added not added in this PR.
But I think this is layer violation.
AFAIK, view specific impls should be in chrome(brave)/browser/ui/views. isn't it?

Reply:
Yeah, I agree. I tried moving it first but it touches chromium_src file so I thought it'd be better to split PRs.

Also, I'm not sure what was the original author's intention. They might have put it under views cause the prefs are only used in views/. Let me trace back what happened.

Reply:
Filed an issue for this: https://github.com/brave/brave-browser/issues/25896